### ISSUE TITLE
Ionic2 app blank screen on iOS at first launch beta11

### ISSUE THREAD

**Issue by vaibsVB** [driftyco/ionic#7638]
Hi guys,

Every time I created a new project in `ionic2` shows `white blank screen` on launch. On `ionic serve` it works fine but on real `iOS device` it shows white screen (this happens with `ionic2 beta-11` too).

below is log generated in `Xcode console`.

>     2016-08-10 18:56:04.670 HelloCordova[401:225916] DiskCookieStorage changing policy from 2 to 0, cookie file: file:///private/var/mobile/Containers/Data/Application/CA19B78E-68F6-4EE9-8358-F5375716B21A/Library/Cookies/Cookies.binarycookies
>     2016-08-10 18:56:04.831 HelloCordova[401:225916] Unable to simultaneously satisfy constraints.
>     	Probably at least one of the constraints in the following list is one you don't want. 
>     	Try this: 
>     		(1) look at each constraint and try to figure out which you don't expect; 
>     		(2) find the code that added the unwanted constraint or constraints and fix it. 
>     (
>         "<NSLayoutConstraint:0x124d2b9e0 V:|-(20)-[UIInputSetContainerView:0x124e56130]   (Names: '|':UITextEffectsWindow:0x124e4fb30 )>",
>         "<NSLayoutConstraint:0x124d325b0 'UIInputWindowController-top' V:|-(0)-[UIInputSetContainerView:0x124e56130]   (Names: '|':UITextEffectsWindow:0x124e4fb30 )>"
>     )

>     Will attempt to recover by breaking constraint 
>     <NSLayoutConstraint:0x124d2b9e0 V:|-(20)-[UIInputSetContainerView:0x124e56130]   (Names: '|':UITextEffectsWindow:0x124e4fb30 )>

>     Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
>     The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
>     2016-08-10 18:56:04.927 HelloCordova[401:225916] Apache Cordova native platform version 4.1.1 is starting.
>     2016-08-10 18:56:04.929 HelloCordova[401:225916] Multi-tasking -> Device: YES, App: YES
>     2016-08-10 18:56:05.005 HelloCordova[401:225916] Using UIWebView
>     2016-08-10 18:56:05.009 HelloCordova[401:225916] [CDVTimer][handleopenurl] 0.220001ms
>     2016-08-10 18:56:05.014 HelloCordova[401:225916] [CDVTimer][intentandnavigationfilter] 3.943026ms
>     2016-08-10 18:56:05.014 HelloCordova[401:225916] [CDVTimer][gesturehandler] 0.187993ms
>     2016-08-10 18:56:05.033 HelloCordova[401:225916] [CDVTimer][splashscreen] 18.348038ms
>     2016-08-10 18:56:05.049 HelloCordova[401:225916] [CDVTimer][statusbar] 16.210973ms
>     2016-08-10 18:56:05.051 HelloCordova[401:225916] [CDVTimer][keyboard] 1.514018ms
>     2016-08-10 18:56:05.051 HelloCordova[401:225916] [CDVTimer][TotalPluginStartup] 42.393982ms
>     2016-08-10 18:56:05.602 HelloCordova[401:225916] Resetting plugins due to page load.
>     2016-08-10 18:56:05.733 HelloCordova[401:225916] Finished load of: file:///var/containers/Bundle/Application/EA59A5E0-FF8B-4BA3-8220-E12D67187C1F/HelloCordova.app/www/index.html

Is anyone facing such issue? And kindly provide a solution for this.

Thanks in advance.

**Comment by vaibsVB** [comment]
I tried `ionic run ios --device` but after completion of build gives this error at the end in terminal.

> 2016-08-10 19:30:47.943 ios-deploy[5713:135452] [ !! ] Error 0xe8008016: The entitlements specified in your application’s Code Signing Entitlements file do not match those specified in your provisioning profile. AMDeviceSecureInstallApplication(0, device, url, options, install_callback, 0)
> 
> Error: Error code 253 for command: ios-deploy with args: --justlaunch,--no-wifi,-d,-b,/Users/bista-mac/Documents/testApp/platforms/ios/build/device/Ionic2TestApp.app

**Comment by jgw96** [comment]
Hey @vaibsVB ! It looks like you have two separate issues here. So for the first issue i have a question, are you storing cookies or using any kind of cookie plugin? Regarding the second error, i would recommend you check out [this](http://blog.ionic.io/deploying-to-a-device-without-an-apple-developer-account/) blog for a detailed walk through of how to get your free ios provisioning setup. Thanks!

**Comment by vaibsVB** [comment]
hey justin,

I toggled the `Keychain sharing` setting in `capabilities` in `Xcode` and assigned particular `Keychain group` i.e. my `free provisioning profile ID`. And it works. Now my project working on iOS real device too.

Thanks

**Comment by jgw96** [comment]
No problem! Thanks for using Ionic!

**Comment by vaibsVB** [comment]
@jgw96 is there any way so that this issue does not generate in future? Because when every time I created a project using `ionic 2 CLI` at first launch it not render on device. Every time I need to add some patch. 

So previously you shared a link that will be the solution or any other setting I have to do. Can you guide me in this?

**Comment by jgw96** [comment]
Unfortunately, there is not much we can do as that is not really an Ionic issue but an Apple xcode issue. I would recommend asking around on our [forum](forum.ionicframework.com) to see if anyone else is having the same issue with their xcode provisioning. I am sorry but I do not fully understand the second question, would you mind reiterating it?

**Comment by vaibsVB** [comment]
Thanks justin for quick reply!

Actually, I'm also not familiar with this type of behaviour of `Xcode`.

And what I found is the bundle identifier of freshly created project is `io.starter.ionic`.

I replaced bundle identifier to `com.demo.testApp` (i.e. what I want to keep it) and after build it utomatically changed to `com.ionicframework.testApp14324`.

This may be issue of `xcode` ofcourse I guess.
